presto-hive/src/main/java/io/prestosql/plugin/hive/util/HiveFileIterator.java

```diff
@@ -108,10 +111,23 @@ protected LocatedFileStatus computeNext()
     private Iterator<LocatedFileStatus> getLocatedFileStatusRemoteIterator(Path path)
     {
         try (TimeStat.BlockTimer ignored = namenodeStats.getListLocatedStatus().time()) {
+            if (ignoreAbsentPartitions && !exists(path)) {
+                return Collections.emptyIterator();
```

Comment:
Nit: static import the method since the name is self-descriptive and thus the `Collections` prefix doesn't provide any value